src/Evolution/Systems/GeneralizedHarmonic/TimeDerivative.cpp

```diff
@@ -300,17 +300,35 @@ void TimeDerivative<Dim>::apply(
   }
 
   // Equation for dt_pi
-  for (size_t mu = 0; mu < Dim + 1; ++mu) {
+
+  // WARNING: normal_dot_gauge_constraint is rescaled by gamma0!
+  get(*normal_dot_gauge_constraint) *= get(gamma0);
```

Comment:
Can you add a comment here explaining what you're doing? It looks like you're avoiding a multiplication of `normal_spacetime_one_form` for the relevant components because it's (1,0,0,0), but that's not immediately obvious.